var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -28,6 +28,7 @@ class Sundials(CMakePackage, CudaPackage, ROCmPackage):
     # Versions
     # ==========================================================================
     version("develop", branch="develop")
+    version("7.1.0", tag="v7.1.0", commit="bacf6acae9008801da209b25a187d7904fd151a3")
```

Comment:
Any particular reason for not using the `sha256` like prior versions?

Reply:
1) `spack checksum` cannot seem to find v7.1.0. 2) This will ensure any pulls via Spack count as a 'clone' in our stats instead of a download.

Reply:
I would like to go update all the older versions to use this approach too, but I will in a different PR.